app/controllers/application_controller.rb

```diff
@@ -1,3 +1,20 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery
+
+  helper_method :logged_in?, :current_user
+
+  def logged_in?
+    !!current_user
+  end
+
+  def current_user
+    current ||= User.find(session[:user_id]) if session[:user_id]
```

Comment:
Thanks @tomtomecek . Didn´t have it quite clear, but after your explanation i have tested in the console and undestand it. One call vs many.